Makefile

```diff
@@ -85,7 +83,7 @@ $(ETH2SPEC): setup.py | $(VENV)
 
 # Force rebuild/install the eth2spec package.
 eth2spec:
-	$(MAKE) --always-make $(ETH2SPEC)
+	@$(MAKE) --always-make $(ETH2SPEC)
```

Comment:
sorry I didn't find it in the previous PR, but here, it seems `eth2spec` command doesn't trigger the rebuild (compile from markdown file to .py files)?

I think we need to trigger the `make pyspec` action: `@$(PYTHON_VENV) setup.py pyspecdev` here, no?

Reply:
These should be separate steps; the generators don't need it. Building eth2spec will compile the markdown files. These will exist in the build & venv dirs. And these are the files which are used by the generators. If I run `make clean` and `make gen_all` it will work without `make pyspec`.

<img width="615" alt="image" src="https://github.com/user-attachments/assets/2620edb9-cd29-4613-876e-e9f26d212582" />

But for `make test` we do need `make pyspec`. It adds these to the `tests` directory.

<img width="615" alt="image" src="https://github.com/user-attachments/assets/14a078d0-576d-49fd-8b58-390ba2f0f3dd" />